### PR TITLE
Implement pulling side icons from server

### PR DESCRIPTION
**Pull request by Plootie**
The icons are a little crispy for some reason. I assume it's due to how the bitmaps are being scaled but I can address that at a later date

### CODE REVIEW

File: SIT.Manager/ViewModels/Play/CharacterSummaryViewModel.cs [hunk 1]
```diff
@@ -52,6 +68,20 @@ public CharacterSummaryViewModel(AkiServer server, AkiMiniProfile profile, ILoca
...
+            if(!string.IsNullOrEmpty(profile.Side) && !profile.Side.Equals("unknown", StringComparison.InvariantCultureIgnoreCase))
+            {
+                string cacheKey = $"side_{Profile.Side} icon";
+                CacheValue<Bitmap> cacheVal = await _cachingService.InMemory.GetOrComputeAsync<Bitmap>(cacheKey, async (key) =>
+                {
+                    return new(await _akiServerRequestingService.GetAkiServerImage(server, $"launcher/side_{profile.Side.ToLower()}.png"));
+                });
+                if(cacheVal.HasValue && cacheVal.Value != null)
+                    SideImage = cacheVal.Value;
+            }
```
Comment (artehe) [inline SIT.Manager/ViewModels/Play/CharacterSummaryViewModel.cs:82]: Would this make more sense to be a function that you just await in the Task run?

File: SIT.Manager/Views/Play/CharacterSummaryView.axaml [hunk 2]
```diff
@@ -11,9 +11,11 @@
...
 			Background="Black">
 		<Grid ColumnDefinitions="Auto, *, Auto">
-			<TextBlock Grid.Column="0"
-					   VerticalAlignment="Center"
-					   Text="{Binding Profile.Side}"/>
+			<Image Grid.Column="0"
+					VerticalAlignment="Center"
+					Source="{Binding SideImage}"
+					MaxWidth="80"
+					MaxHeight="80"/>
```
Comment (artehe) [inline SIT.Manager/Views/Play/CharacterSummaryView.axaml:18]: If you set Stetch="Uniform" then I think that might help the images be clearer

Comment (Plootie) [reply SIT.Manager/Views/Play/CharacterSummaryView.axaml]: I *might* have slightly copied SPT's launcher and it looks mildly better? Idk exactly why their scaling looks smother compared to ours

**Review by artehe — APPROVED** [review]
All good from me, we can fix the todos later with all the others.

**Pull request merged**